Documentation/docs/get-started/Overview.md

```diff
@@ -4,14 +4,10 @@ This pages explains the most important terms and elements of the AsTeRICS framew
 
 ## Solutions
 
-The AsTeRICS framework provides some selected [solutions](/solutions/) which can be used out of the box. The solutions are grouped by **use cases** and can be directly started from the solutions page. Some solutions depend on certain [input devices](#input-device) or other hardware and software. Just click onto the ```Read More``` button in a solution's tile to get a detailed step by step instruction of how to install and use it.
+The AsTeRICS framework provides some selected [solutions](/solutions/) which can be used out of the box. The solutions are grouped by **use cases** and can be directly started from the solutions page. Some solutions depend on certain [input devices](#input-device) or other hardware and software. Just click on the ```Read More``` button in a solution's tile to get a detailed step by step instruction of how to install and use it.
 
 A solution can be categorized by the AsTeRICS technology used. It can be a [model](#model), a [grid](#grid) or an [AT solution](#at-solution).
 
-## Terms
```

Comment:
why an own section `terms` after there was already explained the first term `Solution`?

Reply:
good question, I thought just because to mention the solutions first.